### PR TITLE
main: Default min requests to 0

### PR DESCRIPTION
**Pull request by gvso**
The previous value of 100 is an issue since it requires that the service gets a lot of requests, especially at the beginning of the rollout, to determine the candidate's health

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,18 +127,12 @@ When running the Release Manager, default configuration values are used. The
 most relevant values are:
 
 - `-healthcheck-offset=30m`: When assessing the candidate's health, the Release
-  Manager will get metrics about the candidate revision from the last 30
-  minutes
-- `-min-requests=100`: The Release Manager expects `100` requests in the past 30
-  minutes (given by `healthcheck-offset`). If no enough requests were made in
-  that interval, no rollout or rollback is performed even if the other metrics
-  show a healthy/unhealthy candidate
```
Comment (ahmetb) [inline README.md:135]: I think we should still specify the default as 0

**Comment by ahmetb** [comment]
I think the goal was to add the defaults explicitly to deployment command?

**Comment by gvso** [comment]
Yes. I went ahead and just pass the same values as the default ones, so the experience in CLI is the same as the deployed one by default. Any thoughts on that?

File: README.md [hunk 2]
```diff
@@ -210,7 +210,7 @@ shell:
         --region=us-central1 \
         --image=gcr.io/$PROJECT_ID/cloud-run-release-manager \
         --service-account=release-manager@${PROJECT_ID}.iam.gserviceaccount.com \
-        --args=-verbosity=debug
+        --args=-verbosity=debug,-healthcheck-offset=30m,-min-requests=0,-max-error-rate=1,-min-wait=30m
```
Comment (ahmetb) [inline README.md:213]: split these into separate --args=

File: README.md [hunk 3]
```diff
@@ -127,18 +127,18 @@ When running the Release Manager, default configuration values are used. The
...
-  that interval, no rollout or rollback is performed even if the other metrics
-  show a healthy/unhealthy candidate
+  Manager will get metrics about the candidate revision from the last 30 minutes
+- `-min-requests=0`: The Release Manager expects this number of requests in the
+  past 30 minutes (given by `healthcheck-offset`). If no enough requests were
+  made in that interval, no rollout or rollback is performed even if the other
+  metrics show a healthy/unhealthy candidate. The default value of `0` means
+  that the number of requests should be ignored when the Release Manager makes
+  a decission about rolling out/back.
 - `-max-error-rate=1`: By default, the only health criteria is a expected max
```
Comment (ahmetb) [inline README.md:137]: would be also good to add `(in percent)` here

File: README.md [hunk 4]
```diff
@@ -127,18 +127,18 @@ When running the Release Manager, default configuration values are used. The
 most relevant values are:
 
 - `-healthcheck-offset=30m`: When assessing the candidate's health, the Release
```
Comment (ahmetb) [inline README.md:129]: it's almost like we should document these bullet points right after deploying with these params.

Comment (gvso) [reply README.md:129]: We could add a link to the section "Default rollout configuration"

**Pull request merged**